xla/stream_executor/rocm/rocm_dnn.cc

```diff
@@ -2311,7 +2328,8 @@ bool MIOpenSupport::DoRnnForwardImpl(
       &model_dims);
   if (!res) {
     LOG(ERROR) << "Invalid parameters for RNN Model";
-    return false;
+    return tsl::errors::Internal(
```

Comment:
The use of `tsl::errors::` is discouraged. Could you use `absl::AbortedError ` instead?